app/controllers/roles_controller.rb

```diff
@@ -177,9 +176,12 @@ def build_person(role)
   end
 
   def permitted_params(role_type = entry.class)
-    permitted_attrs = role_type.used_attributes 
-    permitted_attrs -= [:deleted_at] unless can?(:destroy, @role)
-    model_params.permit(permitted_attrs)
+    @permitted_params ||=
+      begin
+        permitted_attrs = role_type.used_attributes
+        permitted_attrs -= [:deleted_at, :delete_on] unless can?(:destroy, @role)
```

Comment:
Ich glaube `deleted_at` sollte jetzt gar nie mehr permitted sein, oder übersehe ich etwas?

Reply:
Es ist denkbar, dass jemand der editieren aber nicht löschen darf den request manipuliert.

Reply:
Ist einfach noch in den `used_attributes` drinnen, wenn dann sollten wir das dort rausnehmen. Macht es Sinn, dass im Zuge von dem Feature noch anzuschauen?